garrysmod/gamemodes/terrortown/gamemode/lang/french.lua

```diff
@@ -0,0 +1,1084 @@
+Utilisez-les pour aider les innocent à survire, mais attention:
+les traitres chercheront à vous tuer en premier!
+
+Appuyez sur {menukey} pour recevoir votre équipement!]]
+
+L.info_popup_traitor_alone = [[Vous êtes un TRAITRE! Vous n'avez pas d'amis traitres ce round.
+
+Tuez tous les autres pour gagner!
+
+Appuyez sur {menukey} pour recevoir votre équipement!]]
+
+L.info_popup_traitor = [[Vous êtes un TRAITRE! Travaillez avec vos amis traitres pour tuer tout le monde.
+Mais faites attention, ou votre trahison pourrait sortir au grand jour...
+
+Voici vos camarades:
+{traitorlist}
+
+Appuyez sur {menukey} pour recevoir votre équipement!]]
+
+--- Various other text
+L.name_kick = "Un joueur a été automatiquement expulsé pour avoir changé son nom pendant un round."
+
+L.idle_popup = [[Vous avez étiez absent {num} secondes et a donc été mis dans le mode Spectateur uniquement. Dans ce mode, vous n'apparaîtrez pas quand un nouveau round démarre.
+
+Vous pouvez basculer hors de ce mode quand vous voulez en appuyant sur {helpkey} et en décochant la case adéquat dans l'onglet Options. Vous pouvez aussi choisir de le désactiver maintenant.]]
+
+L.idle_popup_close = "Ne rien faire"
+L.idle_popup_off   = "Désactiver le mode maintenant"
+
+L.idle_warning = "Attention: on dirait que vous n'êtes plus là, vous allez être déplacé vers les spectateurs à moins de montrer de l'activité!"
+
+L.spec_mode_warning = "Vous êtes en mode Spectateur et vous n'apparaîtrez pas quand un round commence. Pour désactivez ce mode, appuyez sur F1, allez à Options et décochez 'Mode Spectateur uniquement'."
+
+
+--- Tips, shown at bottom of screen to spectators
+
+-- Tips panel
+L.tips_panel_title = "Astuces"
+L.tips_panel_tip   = "Astuce:"
+
+-- Tip texts
+
+L.tip1 = "Les traitres peuvent fouillez un corps silencieusement, sans confirmer la mort, en maintenant {walkkey} et en pressant {usekey} sur le corps."
+
+L.tip2 = "Amorcer un explosif C4 avec un minuteur plus long va augmenter le nombre de fils qui va cause une explosion imminente quand un innocent essaiera de la désamorcer. L'explosif beepera plus doucement et à moins grande fréquence."
+
+L.tip3 = "Les Détectives peuvent fouiller un corps pour trouver qui est 'reflété dans ses yeux'. C'est la dernière personne que le mort a vu. Ce n'est pas forcement le tueur si le mort a été tué dans le dos."
+
+L.tip4 = "Personne ne saura que vous êtes mort jusqu'à ce qu'ils trouvent votre cadavre et vous identifie en le fouillant."
+
+L.tip5 = "Quand un Traitre tue un Détective, ils reçoivent instantanément un crédit."
+
+L.tip6 = "Quand un Traitre meure, tous les Détectives sont récompensés d'un crédit d'équipement."
+
+L.tip7 = "Quand les Traitres ont bien avancé pour tuer les innocents, ils recevront un crédit d'équipement comme récompense."
+
+L.tip8 = "Les Traitres et les Détectives peuvent prendre les crédits d'équipement non-dépensés des corps morts d'autres Traitres et Détectives."

```

Comment:
"non-dépensé" not "non-dépensé**s**"

Reply:
Attends non, c'est LES crédits qui sont non-dépensés

Reply:
Oups, je pensais que le sujet était équipement. My bad.

Reply:
Haha c'est pas grave, t'as déjà fait pas mal :)